src/test/java/seedu/address/model/person/DebtTest.java

```diff
@@ -0,0 +1,52 @@
+package seedu.address.model.person;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.Assert.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+public class DebtTest {
+    private Debt test1 = new Debt("5");
+    private Debt test2 = new Debt("4");
+
+    @Test
+    public void constructor_null_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> new Debt((String) null));
+    }
+
+    @Test
+    public void constructor_invalidDebt_throwsIllegalArgumentException() {
+        String invalidDebtString = "1.001";
+        assertThrows(IllegalArgumentException.class, () -> new Debt(invalidDebtString));
+    }
+
+    @Test
+    public void isValidDebt() {
```

Comment:
Should follow test method names for here, `add()` and `subtract()` command below